app/views/games/show.html.erb

```diff
@@ -8,11 +9,12 @@
   </div>
 </div>
 <table class="chessboard">
-  <% (0..7).each do |row| %>
```

Comment:
board was displaying with black on the bottom. switched around to put white on bottom. HOWEVER, we probably could just switch this line back and forth based on if it's white or black turn, so that the current turn's user always has their pieces on the bottom of the board....